### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by r6eve**
I fixed minor grammar mistakes:

- `every` takes singular nouns.
- `each` takes singular nouns.
- `an` follows the word starts with a vowel sound.
- `depend` requires the preposition `on`.
- Superlative takes `the`.
- The verb cannot be inflected when `do` is already inflected.
- It needs a `,` after a conjunctive adverb.

### CODE REVIEW

File: autoload/vital/_iced/Data/Dict.vim [hunk 1]
```diff
@@ -72,7 +72,7 @@ function! s:swap(dict) abort
   return s:make(values(a:dict), keys(a:dict))
 endfunction
 
-" Makes a index dict from a list
+" Makes an index dict from a list
```
Comment (liquidz) [inline autoload/vital/_iced/Data/Dict.vim]: This file is installed by vital.vim.
So could you send a PR to the following repository?
https://github.com/vim-jp/vital.vim

File: autoload/vital/_iced/Data/List.vim [hunk 2]
```diff
@@ -107,7 +107,7 @@ function! s:concat(list) abort
   return memo
 endfunction
 
-" Take each elements from lists to a new list.
+" Take all elements from lists to a new list.
```
Comment (liquidz) [inline autoload/vital/_iced/Data/List.vim]: Same here.

**Comment by r6eve** [comment]
Thanks for the review. I understand. I'll send another PR later.
I removed it from this PR and squashed the commits.

**Comment by liquidz** [comment]
Thank you so much!

**Pull request merged**